lib/lib-search/src/test/java/org/sagebionetworks/search/awscloudsearch/SynapseToCloudSearchFieldTest.java

```diff
@@ -0,0 +1,43 @@
+package org.sagebionetworks.search.awscloudsearch;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+import java.io.IOException;
+import java.util.List;
+
+import com.amazonaws.services.cloudsearchv2.model.IndexField;
+import org.junit.Test;
+import org.sagebionetworks.repo.model.search.query.SearchFieldName;
+
+public class SynapseToCloudSearchFieldTest {
+
+	@Test
+	public void testLoadSearchDomainSchema() throws IOException{
+		List<IndexField> list = SynapseToCloudSearchField.loadSearchDomainSchema();
+		assertNotNull(list);
```

Comment:
process names into a set and verify that the fields expected in the list are present